### PR TITLE
Enable fully-local development

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
## Solution

When this project was initially started, we were using a local Postgres database and interacting with it via Prisma; however, we've since moved on from Prisma (#). We need some way to continue local DB development. This PR enables that by configuring a locally-hosted instance of supabase.

## Testing

1. Start supabase, `supabase start` (note that supabase credentials are written to stdout).
2. Login to the supabase studio
3. Create tables using the `db.sql` script
2. Update `env.local` with values emitted from supabase CLI
3. Start the app, `yarn dev`
4. Sign in to the app
5. Retrieve your email from the [Inbucket URL](http://localhost:54324/monitor)
6. Click the email link to login

If the above works, you are reasonably assured that this is working as intended. 

_Note you will need docker running on your machine._

### Links

* [Hosting Supabase locally](https://supabase.com/docs/guides/local-development)
* [Viewing emails sent from locally-hosted supabase instance](https://github.com/supabase/cli/issues/93#issuecomment-989991510)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+⚠️ Click preview button above ⚠️
+
+# PR Template
+
+1. Apply relevant labels.
+1. Self-review code and PR.
+1. Request review from a team member.
+1. Wait for the code review.
+1. Take the bits that apply to your PR and delete the rest of template.
```
Comment (Yohanan-Negash) [inline .github/pull_request_template.md:9]: Are these tasks put in order? If not, is there a sequence here that we should follow?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Call out changes that may lead to deployment failure
+  - new or updated test specs
+  - dependency and other package updates (use label and link to dependency docs)
+  - new migrations (use label)
+  - devOps changes (use label)
+
+```md
+###### Thing your're calling out
+
+The thing being called out. Use a list when it makes sense.
```
Comment (Yohanan-Negash) [inline .github/pull_request_template.md:26]: What thing is being called out here? Can you elaborate on this?

**Review by Yohanan-Negash — COMMENTED** [review]
The code looks fine, but there are two minor issues that are worth reviewing in the `.md`  file. I listed them out in the review above.

File: .github/pull_request_template.md (hunk 1 above)
Comment (iAmWillShepherd) [reply .github/pull_request_template.md:9]: Yes, they are in order.

File: .github/pull_request_template.md (hunk 2 above)
Comment (iAmWillShepherd) [reply .github/pull_request_template.md:26]: It's up to the author, I can't say here.

**Comment by iAmWillShepherd** [comment]
I merged this before your review. The things you called out are none-issues; the order is correct, and the section stating to call out stuff depends on the work introduced in the PR.

**Pull request merged**